ovn_cluster.sh

```diff
@@ -484,13 +587,15 @@ function start() {
 
     # Create containers
     if [ "$ovn_central" == "yes" ]; then
-        if [ "$OVN_DB_CLUSTER" = "yes" ]; then
-            start-container "${CENTRAL_IMAGE}" "${CENTRAL_NAME}-1"
-            start-container "${CENTRAL_IMAGE}" "${CENTRAL_NAME}-2"
-            start-container "${CENTRAL_IMAGE}" "${CENTRAL_NAME}-3"
-        else
-            start-container "${CENTRAL_IMAGE}" "${CENTRAL_NAME}"
-        fi
+        for name in "${CENTRAL_NAMES[@]}"; do
+            if [ "$OVN_DB_CLUSTER" = "yes" ]; then
+                start-container "${CENTRAL_IMAGE}" "${name}-1"
+                start-container "${CENTRAL_IMAGE}" "${name}-2"
+                start-container "${CENTRAL_IMAGE}" "${name}-3"
+            else
+                start-container "${CENTRAL_IMAGE}" "${name}"
```

Comment:
If we change this to `start-container "${CENTRAL_IMAGE}" "${name}"-1` then the first DB container always has the same name regardless of clustered DBs or not.  That simplifies a bit the logic in other places.

Reply:
@dceara ack, I will fix it